app/controllers/users_controller.rb

```diff
@@ -81,14 +81,21 @@ def new # TODO needs more rigorous testing
   def add_patient
     current_user.add_patient @patient
     respond_to do |format|
-      format.js { render template: 'users/refresh_patients', layout: false }
+      format.js { render template: 'users/refresh_patients', locals: {msg: 'Successfully added Patient'}, layout: false }
     end
   end
 
   def remove_patient
     current_user.remove_patient @patient
     respond_to do |format|
-      format.js { render template: 'users/refresh_patients', layout: false }
+      format.js { render template: 'users/refresh_patients', locals: {msg: 'Successfully removed Patient'}, layout: false }
+    end
+  end
+
+  def remove_all_patients
```

Comment:
Thoughts about changing the name of this to `clear_call_list`? I think that might be a little closer to actual behavior -- your thoughts?